run_cart.sh

```diff
@@ -24,6 +24,7 @@ CARTDIR="/home1/<PATH_TO_CART>/cart"
 
 interface=ib0
 sep=0
+sep_cart=0
```

Comment:
I think the name sep_cart here can be misleading here since it doesn't really reflect the use of scalable endpoint. I think a more effective way of naming the folder would be to reflect the characteristic of binary being run. This will also make looking up logs easier. We can pass this info to the functions run_bg, run_ft, run_st. 
Example of names: test_group_srv, selftest_inf<x>_nserver<y>, etc
where inf is the max_inflight and nserver is the number of server

Reply:
Yes that should be idle, I was just trying to fix the issue and get the logs for all commands.